sympy/series/formal.py

```diff
@@ -810,6 +811,11 @@ def _compute_fps(f, x, x0, dir, hyper, order, rational, full):
             return ak, xk, ind
         return None
 
+    syms = list(f.free_symbols.difference({x}))
```

Comment:
Can we have a ``free_symbols`` property instead? I am not sure but I think it should work in general. Can you check?

Reply:
I didn't get you. Could you elaborate?

Reply:
https://docs.sympy.org/latest/modules/core.html#sympy.core.basic.Basic.free_symbols

Reply:
I have read that. But I couldn't understand which property you are mentioning? Can you tell that? There are no properties of `free_symbols` given in the above link.

Reply:
> I have read that. But I couldn't understand which property you are mentioning? Can you tell that? There are no properties of `free_symbols` given in the above link.

Can you please recheck.